lib/screens/v2/screen_data/query_params.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Screens.V2.ScreenData.QueryParams do
+  @moduledoc """
+  Encodes valid query parameters that are currently only used by on bus screens.
+  Contains methods for processing query paramaters used by screen and api controllers.
```

Comment:
The bit here about "screen and api controllers" doesn't seem accurate anymore. (in general I try to avoid doc comments that assert things about where/how the documented entity is currently used, because they inevitably fall out of date)

Reply:
that's a good philosophy! I did leave in the comment about it only being used for bus screens right now, b/c if it's updated again in the future that point seems very relevant to remember